app/pages/sections/order-items/additional-services/select/price/controller.js

```diff
@@ -1,24 +1,8 @@
-import { getData } from 'buying-catalogue-library';
 import { getContext, getErrorContext } from './contextCreator';
-import { getEndpoint } from '../../../../../../endpoints';
-import { logger } from '../../../../../../logger';
 
 export const getAdditionalServicePricePageContext = params => getContext(params);
 export const getAdditionalServicePriceErrorPageContext = params => getErrorContext(params);
 
-export const findAdditionalServicePrices = async ({ accessToken, catalogueItemId }) => {
-  const additionalServicePricingEndpoint = getEndpoint({ api: 'bapi', endpointLocator: 'getCatalogueItemPricing', options: { catalogueItemId } });
-  const additionalServicePricingData = await getData({
-    endpoint: additionalServicePricingEndpoint,
-    accessToken,
-    logger,
-  });
-
-  logger.info(`Additional service pricing for additional service with ID: ${catalogueItemId} found in BAPI.`);
-
-  return additionalServicePricingData;
-};
-
 export const validateAdditionalServicePriceForm = ({ data }) => {
```

Comment:
Nice.. is there one in catalogue-solutions/select/price that can also be removed

Reply:
nah that uses getSolutionPricing

Reply:
ok I think that can be changed to use this too. But leave as is for now. I'll take a look later 👍